examples/demo_expectations.pck

```diff
@@ -85,15 +85,15 @@ create or replace package body demo_expectations is
     ut.expect( to_char(null), 'this should fail' ).not_to( equal( to_char(null) ) );
 
     --fails on nulls not beeig equal
-    ut_assert_processor.nulls_are_equal(false);
+    ut.nulls_are_equal(false);
```

Comment:
Hmm...
This procedure was actually meant to become an annotation.
It would be much better if we could keep the `null_are_equal` contained in a way we have it with transaction handling etc.
So that when you annotate a suite or test with
`-- %nulls_equal`
The scope of the change is on a whole test or whole suite level.

I think the null_are_equal as a procedure should not be exposed to users because, when you set it in one tests it affects other tests unless you reset it back to default.

If we would agree to have such annotation then for this PR I'd suggest one of two things:
- comment the tests that use that functionality until we have annotation
- add new annotation support as part of this PR

I guess the firs one seems more appropriate, so we do not have too many changes in a PR.

What do you think?